doc/source/whatsnew/v1.0.0.rst

```diff
@@ -190,6 +190,12 @@ ExtensionArray
 -
 -
 
+
+Other
+^^^^^
+- Trying to set the display.precision, display.max_rows or display.max_columns using set_option to anything but a None or a positive int will raise a ``ValueError`` (:issue:`23348`)
```

Comment:
Put the option names like `display.precision` in double-backticks.

set_option -> :func:\`pandas.set_option\` 

a None -> `None`

int -> integer.